### DISCUSSION TITLE
docs: Improve "Installation & Usage" documentation

### DISCUSSION THREAD

**Discussion by henriqueleite42** [mikro-orm/mikro-orm#4951] [General]
Hi!

I'm just discovered mikro-orm, I haven't used it yet, but loved the whole idea and I'm trying to implement it in a project to start testing. During this process, I noticed something that I think would be nice to have in the docs.

## From where `orm` and `MikroORM` come from?
![image](https://github.com/mikro-orm/mikro-orm/assets/50992814/ecfd3106-9cc8-4440-ae10-1c30f406759b)

Trying to implement it and reading a bit more of the docs, I can assume that `orm` comes from `await MikroORM.init`, and `MikroORM` is exported both by `@mikro-orm/core` and `@mikro-orm/postgresql`, and as I'm using Postgres, I guess that I have to import it from `@mikro-orm/postgresql`, right?

For the `orm`, I think that just changing the order of the 2 examples would be enough to don't cause any confusion, and for the `MikroORM` I think that a more detailed explanation about where to import it would be great.

If you guys agree with this change, I would be happy to create a PR to make these changes or any changes that you guys consider useful.

**Comment by B4nan** [comment]
Check out the next docs, they are already greatly improved, including the new getting started guide as well as polished quick start.

https://mikro-orm.io/docs/next/quick-start

I don't mind any improvements, feel free to send a PR to make things clearer, just don't expect me to merge things right ahead (more often then not, people send a PR and never address any feedback on it).
